docs/source/developers/contributing.rst

```diff
@@ -78,6 +78,9 @@ assign yourself an issue, you will need "Contributor" permissions on the
 Apache Arrow JIRA. To get this role, ask on the mailing list for a project
 maintainer's help.
 
+
+.. _jira-tips:
+
 Tips for using JIRA
```

Comment:
Should there be a "Creating a JIRA account" section just before this? You could move the text you've already written in "Finding good first issues" there.

Reply:
Not sure I understood completely. Will rearrange the text and see if that's what u meant.

Reply:
Two paragraphs above already has a short explanation of needing to sign up to JIRA